crates/ruma-common/src/events/room/aliases.rs

```diff
@@ -49,6 +49,11 @@ impl RedactContent for RoomAliasesEventContent {
     }
 }
 
+/// The possibly redacted form of [`RoomAliasesEventContent`].
+///
+/// This type is used when it's not obvious whether the content is redacted or not.
+pub type PossiblyRedactedRoomAliasesEventContent = RedactedRoomAliasesEventContent;
```

Comment:
I think this this should be a separate type, in case extra fields that don't survive redaction are added to this event.